doc/api/documentation.md

```diff
@@ -32,6 +32,17 @@ The stability indices are as follows:
 > [semantic versioning][] rules. Non-backward compatible changes or removal may
 > occur in any future release. Use of the feature is not recommended in
 > production environments.
+>
+> Experimental features are subdivided into stages:
+>
+> * 1.0 - Early development. Experimental features at this stage are unfinished
+>   and subject to substantial change.
+> * 1.1 - Active development. Experimental features at this stage are nearing
+>   potential stability but breaking changes are likely.
```

Comment:
This seems to have 2 conflicting things - "potential stability" and "likely breaking changes". I'm not able to understand how something can be both at the same time.

Reply:
Basically, _we_ think the feature is ready, but if users prove us wrong—they point out something we missed, that might be a bigger oversight than just a bug—then a breaking change will still happen. That’s the whole point of the stage, that this is our last chance to make breaking changes before marking something stable. We _hope_ there won’t be any breaking chances, we aren’t _planning_ on any, but you never know. This is essentially the QA pass.